### DISCUSSION TITLE
What is the path forward for this runtime to be considered non-experimental?

### DISCUSSION THREAD

**Discussion by zakhenry** [aws/aws-lambda-rust-runtime#920] [General]
The readme states
> The Rust runtime client is an experimental package. It is subject to change and intended only for evaluation purposes.

What needs to happen for this runtime to be a v1? Is there any kind of timeline known?

**Comment by bassmanitram** [comment]
Similar question - we have a bunch of stuff just waiting to go - the blocker is that "experimental" note.

**Comment by zakhenry** [comment]
@jlizen apologies for the ping, but I’m hoping you might have some insider knowledge that could be shared?

I think many like me are withholding using this repo for anything that’s not an experiment, so it would be useful to know what the process is to unlock all the general purpose use cases we want to use rust with lambda for. 

Perhaps it’s already in a good stable state and we can just remove that qualifier? 🤞

**Comment by jlizen** [comment]
This would be a better question for @PartiallyUntyped as they work in the AWS Lambda org.

Declaring the runtime as "prod ready" is more of a business decision for Lambda regardless of how solid/stable the runtime's code is (pretty solid, I feel). Since it involves officially supporting it, etc.

Anything we can share, Astraea, around a prospective path to GA for the rust runtime?

**Comment by ghost** [comment]
Unfortunately I can't offer much other than highlighting that two engineers from the runtimes team have been contributing lately. 🤞

**Comment by zakhenry** [comment]
Well done everyone involved getting this to GA 🎉

**Comment by jtuliani** [comment]
For completeness, here's the What's New post announcing Lambda Rust support promotion from 'Experimental' to 'Generally Available': https://aws.amazon.com/about-aws/whats-new/2025/11/aws-lambda-rust/
